viam_cartographer.go

```diff
@@ -337,16 +341,24 @@ func parseCartoAlgoConfig(configParams map[string]string, logger logging.Logger)
 			}
 			cartoAlgoCfg.MissingDataRayLength = float32(fVal)
 		case "max_range":
+
 			fVal, err := strconv.ParseFloat(val, 32)
 			if err != nil {
 				return cartoAlgoCfg, err
 			}
 			cartoAlgoCfg.MaxRange = float32(fVal)
 		case "max_range_meters":
```

Comment:
only making this change for `max_range_meters` and `min_range_meters` because they are the only parameters that automatically get added to the config on the r2d2 slam card with the chance of being empty